commit 3a21a94e982132266eaa99de51906a6916a484bb (refs/changes/91/4591/2)
Author: Trent Mick <trentm@gmail.com>
Date:   2018-07-26T17:35:26-07:00 (1 year, 2 months ago)
    
    TRITON-635 restify-clients usage should use 'contentMd5' option to avoid cross-node version BadDigest errors
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/CHANGES.md b/CHANGES.md
index bfb6114..a880a75 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -4,6 +4,10 @@
 
 (nothing yet)
 
+## 0.3.1
+
+- TRITON-635 restify-clients usage has been updated to use the new 'contentMd5'
+  option to avoid cross-node version BadDigest errors.
 
 ## 0.3.0
 
diff --git a/README.md b/README.md
index b81f459..cd40477 100644
--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@ See [the wf-client docs here](./docs/index.md).
 
     make test
 
-Currently this assumes a Workflow API endpoint at <https://10.99.99.19>, the
+Currently this assumes a Workflow API endpoint at <http://10.99.99.19>, the
 common IP for a [CoaL setup](https://github.com/joyent/triton#getting-started).
 To test a workflow API at a different IP run:
 
diff --git a/lib/wf-client.js b/lib/wf-client.js
index df96536..3090871 100644
--- a/lib/wf-client.js
+++ b/lib/wf-client.js
@@ -28,7 +28,10 @@ function WfClient(options) {
     var restifyOptions = {
         url: options.url,
         version: '*',
-        log: options.log
+        log: options.log,
+        contentMd5: {
+            encodings: ['utf8', 'binary']
+        }
     };
 
     if (options.agent !== undefined) restifyOptions.agent = options.agent;
diff --git a/package.json b/package.json
index fbe13d0..7339879 100644
--- a/package.json
+++ b/package.json
@@ -1,7 +1,7 @@
 {
   "name": "wf-client",
   "description": "a node.js client for the Triton core Workflow API service",
-  "version": "0.3.0",
+  "version": "0.3.1",
   "author": "Joyent (joyent.com)",
   "main": "./lib/wf-client.js",
   "dependencies": {
@@ -9,7 +9,7 @@
     "async": "~0.9.0",
     "backoff": "^2.4.1",
     "bunyan": "^1.8.1",
-    "restify-clients": "^1.3.1"
+    "restify-clients": "^1.6.0"
   },
   "devDependencies": {
     "nodeunit": "^0.10.2"
